tests/docker/build.sh

```diff
@@ -70,8 +72,9 @@ cargo cinstall --prefix=/usr --release
 cd
 git clone -b v0.9.1 --depth 1 https://gitlab.com/AOMediaCodec/SVT-AV1.git
 cd SVT-AV1
-cd Build/linux
-./build.sh --prefix=/usr release install
+cd Build
+cmake -G Ninja -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release ..
+ninja install
```

Comment:
@1480c1 @hassount Please review. Thanks.

I modeled after the commands at https://gitlab.com/AOMediaCodec/SVT-AV1#1-build-and-install-svt-av1.